File: Modul 4/Soal 5/PRAK305-2210817220018-Piola Evania.c
#include <stdio.h>

int main()
{
    for(int i=1; i<6; i++){
    int s, sec, mnt, hr, d;
    scanf("%d", &s);
    sec = s%60;
    mnt = (s/60)%60;
    hr = (s/3600)%24;
    d = s/86400;

    if(s<=3600){
        printf("%.2d:%.2d:%.2d\n\n", hr, mnt, sec);
    }
    else if(s>3600 && s<86400){
        printf("%.2d:%.2d:%.2d\n\n", hr, mnt, sec);
    }
    else{
        printf("%d hari %.2d:%.2d:%.2d\n\n", d, hr, mnt, sec);
    }
    }
}

File: Modul 4/Soal 5/PRAK305-2210817220018-Piola Evania.py
for i in range(5):   
    s = int(input())
    sec = s%60
    mnt = (s/60)%60
    hr = (s/3600)%24
    d = s/86400

    if (s<=3600):
        print("%.2d:%.2d:%.2d\n" %(hr, mnt, sec))
    elif (s>3600 and s<=86400):
        print("%.2d:%.2d:%.2d\n" %(hr, mnt, sec))
    else:
        print("%d hari %.2d:%.2d:%.2d\n" %(d, hr, mnt, sec))
